### ISSUE TITLE
Question about output

### ISSUE THREAD

**Issue by yaninsanity** [Ajay2810-hub/directory-tree-generator#2]
Hey this lib is very helpful to my project. Quick question here, can we add an interface to output the tree as a md file?

**Comment by Ajay2810-hub** [comment]
Yes sure that would be nice.

**Comment by ajay-lingayat** [comment]
> Hey this lib is very helpful to my project. Quick question here, can we add an interface to output the tree as a md file?

 Yes sure that would be great. If you have already worked it out then please raise a PR.
